README.md

```diff
@@ -84,6 +84,13 @@ download and unarchive the source tarball (Appium-Python-Client-X.X.tar.gz).
     python setup.py install
     ```
 
+> **Note**
+> The Appium Python Client depends on [Selenium Python binding](https://pypi.org/project/selenium/), thus
+> the Selenium Python binding update might affect the Appium Python Client.
+> For exampple, some changes in the Selenium binding could break the Appium client.
+> To reduce such unexpected behavior, we strongly recommend you to manage dependencies
```

Comment:
We probably need to explicitly point out the source for a compatible selenium version. Perhaps, maintaining a table like https://github.com/appium/java-client#compatibility-matrix would also make sense

Reply:
Sounds good. need to recall past versions, so let me prepare as another pr

Reply:
-> https://github.com/appium/python-client/pull/876
The version mapping is simpler than Java client's dependencies, so it currently only has a matrix table.

Reply:
Moved into this pr